.eslintrc.js

```diff
@@ -1225,6 +1225,7 @@ module.exports = {
                 extensions: ['.tsx']
             }
         ],
+        'react/jsx-uses-vars': 'error',
```

Comment:
I was getting hit with this warnings in my last PR, so I had disabled them by file. Turns out there is a setting here when I went to dig in deeper:
https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/jsx-uses-vars.md